README.md

```diff
@@ -0,0 +1,20 @@
+# OneKey Wallet Flatpak
+
+Unofficial [Flathub package](https://flathub.org/apps/details/so.onekey.Wallet) of [OneKey Wallet](https://onekey.so/).
+
+You need to install **PCSC daemon** to use this app.
```

Comment:
Users are unlikely to ever see your README. You need to mention this requirement in the appdata file.

Reply:
Up to you. Just pointing out that README is not user-facing.